File: Programming/Practice/21/Python/21.py
def bmi(weight: float, height: float):
    bmi = weight / height**2
    return float(bmi)
def print_bmi(bmi: float):
    if bmi < 18.5:
        print("Underweight")
    if bmi >= 18.5 and bmi < 25:
        print("Normal")
    if bmi >= 25 and bmi < 30:
        print("Overweight")
    if bmi >= 30:
        print("Obesity")
    return
print("Введите значение веса и роста:")
ves, rost = map(float,input().split())
bmi = bmi(ves, rost)
print_bmi(bmi)

File: Programming/Practice/21/C++/21/21/21.cpp
#include<iostream>
using namespace std;
double BMI(double weight, double height)
{
	double BMI = (weight / (height * height));
	return BMI;
}

void printBMI(double BMI)
{
	if (BMI < 18.5) cout << "Underweight" << endl;
	if ((18.5 <= BMI) && (BMI < 25)) cout << "Normal" << endl;
	if ((25 <= BMI) && (BMI < 30)) cout << "Overweight" << endl;
	if (30 <= BMI) cout << "Obesity" << endl;
}

int main()
{
	setlocale(LC_ALL, "Rus");
	double weight, height, bmi;
	cout << "Введите вес и рост:" << endl;
	cin >> weight >> height;

	bmi = BMI(weight, height);

	printBMI(bmi);


}